executor/analyze.go

```diff
@@ -190,6 +191,8 @@ func (e *AnalyzeExec) analyzeWorker(taskCh <-chan *analyzeTask, resultCh chan<-
 			task.job.Start()
 			resultCh <- analyzeIndexPushdown(task.idxExec)
 		case fastTask:
+			task.fastExec.job = task.job
+			task.job.Start()
```

Comment:
Do we need `AnalyzeJob::Update` for fast analyze?

Reply:
Added `Update`.